layout/_third-party/copy-code.swig

```diff
@@ -45,19 +45,27 @@
           return $(e).text()
         }).toArray().join('\n')
         var ta = document.createElement('textarea')
-        document.body.appendChild(ta)
-        ta.style.position = 'absolute'
-        ta.style.top = '0px'
-        ta.style.left = '0px'
+        var range = document.createRange() //For Chrome
```

Comment:
The original method is kind of ... tricky? What I tried to do is finding a general solution works on most modern browsers. And because `document.createRange` has different behaviour in chromium & firefox, extra code is needed.